app/views/publishers/vacancies/build/_steps.html.slim

```diff
@@ -0,0 +1,4 @@
+= render StepsComponent.new title: t("publishers.vacancies.review.steps"), classes: "govuk-!-margin-top-6" do |component|
+  - steps_to_display(steps_config).each do |vacancy_step, step_number|
+    - component.step(label: t("publishers.vacancies.build.#{vacancy_step}.step_title"), current: (vacancy_step == step_current), completed: vacancy_step_completed?(vacancy, step_number))
+  - component.step(label: t("publishers.vacancies.review.heading"), current: step_current == :review, completed: false)
```

Comment:
`completed: false`

Is this partial shown if we go to edit an already published job listing?

In that case should the review step be shown as complete?

Reply:
It is shown if we edit an already published job listing, but it shouldn't be. I'll change that now.